File: Code_ground/NO_0111/111.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 07                 | #
# | Day 25                   | #
# | 파티셰리                   | #
# ---------------------------- #


if __name__ == "__main__":
    T = int(r_input())

    for testcase in range(1, T + 1):
        N = int(r_input())

        d, c, m = 11, 9, 7

        res = 'NO'

        for ind1 in range(100 // d):
            for ind2 in range(100 // c):
                for ind3 in range(100 // m):
                    if ind1 * d + ind2 * c + ind3 * m == N:
                        res = 'YES'
                        break
                
                if res == 'YES':
                    break
            
            if res == 'YES':
                break
        
        print('Case #' + str(testcase))
        print(res)

File: Code_ground/NO_0111/111.cpp
#include <iostream>
using namespace std;
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 28                   | 
 | 파티셰리                   | 
 ---------------------------- 
*/

int main(){
    int T;

    cin >> T;
    
    for(int testcase = 1; testcase <= T; testcase ++){
        int N, flag = 0;

        cin >> N;

        for(int i = 0; i < N / 11 + 1; i ++){
            for(int j = 0; j < N / 9 + 1; j ++){
                for(int k = 0; k < N / 7 + 1; k ++){
                    if(i * 11 + j * 9 + k * 7 == N){
                        flag = 1;
                        break;
                    }
                }

                if(flag) break;
            }

            if(flag) break;
        }

        cout << "Case #" << testcase << endl;
        if(flag) cout << "YES" << endl;
        else cout << "NO" << endl;
    }

    return 0;
}
